src/terminal/terminal/display.h

```diff
@@ -369,5 +380,16 @@ void guac_terminal_display_clear_select(guac_terminal_display* display);
 int guac_terminal_display_set_font(guac_terminal_display* display,
         const char* font_name, int font_size, int dpi);
 
+/**
+ * Calculate the size of margins around the terminal based on DPI.
+ *
+ * @param dpi
+ *     The resolution of the display in DPI.
+ *
+ * @return
+ *     Calculated size of margin in pixels
+ */
+int get_margin_by_dpi(int dpi);        
+
```

Comment:
I don't see this function used anywhere outside of `display.c` - is there some reason it should be made available outside of the file, or perhaps it should be a `static inc` within `display.c`?